include/mxnet/c_api.h

```diff
@@ -1530,6 +1530,18 @@ MXNET_DLL int MXInitPSEnv(mx_uint num_vars,
  */
 MXNET_DLL int MXKVStoreCreate(const char *type,
                               KVStoreHandle *out);
+
+/*!
+ * \brief Set parameters to use low-bit compressed gradients
+ * \param handle handle to the kvstore
+ * \param compression type of compression
+ * \param threshold set the threshold for 2bit compression
+ * \return 0 when success, -1 when failure happens
+ */
+MXNET_DLL int MXKVStoreSetGradientCompression(KVStoreHandle handle,
+                                   const char *compression,
+                                   const float threshold);
```

Comment:
API should be
`MXKVStoreSetGradientCompression(KVStoreHandle handle,
mx_uint num_params,
const char **keys,
const char **vals)`
The values should be parsed in backend with `dmlc::Parameter`